core/config/locales/en.yml

```diff
@@ -170,13 +170,14 @@ en:
         code: Code
         description: Description
         event_name: Event Name
-        expires_at: Expiration date
+        expires_at: End
         name: Name
         path: Path
         per_code_usage_limit: Per Code Usage Limit
         promotion_uses: Promotion Uses
-        starts_at: Start date
-        usage_limit: Overall Usage Limit
+        starts_at: Start
+        status: Status
+        usage_limit: Usage Limit
```

Comment:
These changes do not belong in this PR IMO. At least it should be a separate commit.